tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -219,5 +248,37 @@ def handler(event: Dict[str, Union[str, int]],
     Raises:
         ExtractFilePathsError: An error occurred parsing the input.
     """
-    result = run_cumulus_task(task, event, context)
+
+    # Generating schema validators can take time, so do it once and reuse.
+    try:
+        with open("schemas/input.json", "r") as raw_schema:
+            input_schema = json.loads(raw_schema.read())
+            _VALIDATE_INPUT = fastjsonschema.compile(input_schema)
+        with open("schemas/config.json", "r") as raw_schema:
+            config_schema = json.loads(raw_schema.read())
+            _VALIDATE_CONFIG = fastjsonschema.compile(config_schema)
+    except Exception as ex:
+        LOGGER.error(f"Could not build schema validator: {ex}")
+        raise
+
+    try:
+        _VALIDATE_INPUT(event["payload"])
+    except JsonSchemaException as json_schema_exception:
+        return create_http_error_dict(
```

Comment:
Per previous comments, I would log and reraise the error. I would not send an HTTP error dict since this is not behind an API GW.

Reply:
fixed